code/eyetracking/schedule.tsv

```diff
@@ -1,5 +1,5 @@
-session	day	PE	fixation_edf	qct_edf	rest_edf	bht_edf
-1	2023-10-20	LR	fixation_2023-10-20_18h48.03.561_5_session_1.EDF	qct_2023-10-20_19h40.38.964_2_session_1.EDF	rest_2023-10-20_19h48.22.344_0_session_1.EDF	bht_2023-10-20_20h10.50.517_0_session_1.EDF
+session	day	PE	dwi_edf	qct_edf	rest_edf	bht_edf
```

Comment:
why this change?

Reply:
It was an error (same for the comment below). I reverted the schedule to the correct version.